zaza/openstack/charm_tests/test_utils.py

```diff
@@ -426,6 +427,50 @@ def pause_resume(self, services, pgrep_full=False):
             model_name=self.model_name,
             pgrep_full=pgrep_full)
 
+    def get_my_tests_options(self, key, default=None):
+        """Retrieve tests_options for specific test.
+
+        Prefix for key is built from dot-notated absolute path to calling
+        method or function.
+
+        Example:
+           # In tests.yaml:
+           tests_options:
+             zaza.charm_tests.noop.tests.NoopTest.test_foo.key: true
+           # called from zaza.charm_tests.noop.tests.NoopTest.test_foo()
+           >>> get_my_tests_options('key')
+           True
+
```

Comment:
I *think* this is a new introduction of tests_options. I'd like a second opinion on this. Let's discuss in the stand-up.

Reply:
This uses the existing tests_options functionality, the problem it solves is that the tests_options provides a flat namespace for configuration keys. I want to use the [module path of the test to avoid collisions](https://review.opendev.org/#/c/740974/18/tests/tests.yaml@68). Instead of hard coding that into the test I add a re-usable helper to do it for me.

Reply:
As discussed the developers agree this works.